CMakeLists.txt

```diff
@@ -27,8 +27,12 @@ target_include_directories(oboe
                            PRIVATE src 
                            PUBLIC include)
 
-target_compile_options(oboe PRIVATE -std=c++11
+ # Compile Flags:
+ #     Enable -Werror when building debug config
+ #     Enable -Ofast for release config (could be enabled for debug too)
+ target_compile_options(oboe PRIVATE -std=c++11
                             PRIVATE -Wall
-                            PRIVATE "$<$<CONFIG:DEBUG>:-Werror>") # Only include -Werror when building debug config
+                            PRIVATE "$<$<CONFIG:DEBUG>:-Werror>"
+                            PRIVATE "$<$<CONFIG:RELEASE>:-Ofast>")
```

Comment:
Oboe is currently built from source when a dependent project (e.g. LiveEffect) is built. Is the build configuration (release or debug) used when building Oboe?